### PR TITLE
Delete abstain option

### PR DESCRIPTION
**Pull request by selankon**
Abstain option and badge are not shown anymore. 

Instead, if abstain is possible, you can just continue selecting only zero or n options. 

A warning message will appear on confirmation modal.

image

Implemented on https://github.com/vocdoni/ui-scaffold/pull/648

If less options than max are chosen, it will add the abstain choices to the ballor

### CODE REVIEW

File: packages/chakra-components/src/components/Election/Questions.tsx [hunk 1]
```diff
@@ -280,14 +279,7 @@ const MultiChoice = ({ index, question }: QuestionProps) => {
...
   const choices = [...question.choices]
-  if (election.resultsType.properties.canAbstain) {
-    choices.push({
-      title: {
-        default: localize('vote.abstain'),
-      },
-      value: -1,
-    })
-  }
+  const canAbstain = election.resultsType.properties.canAbstain
```
Comment (elboletaire) [inline packages/chakra-components/src/components/Election/Questions.tsx:282]: Why define a variable if it's only used once? I guess you wanted to define it before, and use it in the condition above? 🤔

**Review by elboletaire — COMMENTED** [review]
Based on the code of this PR, I don't understand what's going on with the abstain option anymore. I mean, I see we now accept sending empty results, but where are the changes for the mapping of these abstains when voting? Or aren't required anymore due to underlying SDK changes? 🤔

File: packages/chakra-components/src/components/Election/Questions.tsx (hunk 1 above)
Comment (selankon) [reply packages/chakra-components/src/components/Election/Questions.tsx]: 84cd1a3

**Comment by selankon** [comment]
> Based on the code of this PR, I don't understand what's going on with the abstain option anymore. I mean, I see we now accept sending empty results, but where are the changes for the mapping of these abstains when voting? Or aren't required anymore due to underlying SDK changes? 🤔

On the SDK anything changed. Now the flow is quite different, and if abstain is enabled the abstain options will be mapped here, during the ballot creation.

https://github.com/vocdoni/ui-components/blob/84cd1a303bd88551e4ed1bce67ca16d09c2f23a0/packages/chakra-components/src/components/Election/Questions.tsx#L89-L94

**Review by elboletaire — APPROVED** [review]

**Pull request merged**